tests/tests_integration/test_api/test_datapoints.py

```diff
@@ -1969,6 +1985,36 @@ def test_calendar_granularities_in_utc_and_timezone(self, retrieve_endpoints, al
             assert_equal(exp_days_per_quarter, q_utc.count[: 4 * 4])
             assert_equal(exp_days_per_month, mo_utc.count[: 12 * 4])
 
+    @pytest.mark.parametrize("include_status", (True, False))
+    def test_object_aggregates(self, retrieve_endpoints, all_numeric_test_time_series, include_status):
+        ts1, ts2 = random.sample(all_numeric_test_time_series, k=2)
+        for endpoint in retrieve_endpoints:
+            dps_lst = endpoint(
+                id=ts1.id,
+                external_id=ts2.external_id,
+                start=MIN_TIMESTAMP_MS,
+                end=MAX_TIMESTAMP_MS,
+                aggregates=list(_OBJECT_AGGREGATES),
+                granularity="1mo",
+                include_status=include_status,
+                limit=2,
+            )
+            for dps in dps_lst:
+                assert 1 <= len(dps.min_datapoint) <= 2
+                assert 1 <= len(dps.max_datapoint) <= 2
+                dp = dps[0]
+                assert isinstance(dp, Datapoint)
+                if include_status:
+                    assert isinstance(dps.min_datapoint[0], MinDatapointWithStatus)
+                    assert isinstance(dps.max_datapoint[0], MaxDatapointWithStatus)
+                    assert isinstance(dp.min_datapoint, MinDatapointWithStatus)
+                    assert isinstance(dp.max_datapoint, MaxDatapointWithStatus)
+                else:
+                    assert isinstance(dps.min_datapoint[0], MinDatapoint)
+                    assert isinstance(dps.max_datapoint[0], MaxDatapoint)
+                    assert isinstance(dp.min_datapoint, MinDatapoint)
+                    assert isinstance(dp.max_datapoint, MaxDatapoint)
```

Comment:
I assume there's not many assertions because the data is random?
At least couple thing to check could be
- numbers aren't NaN
- min <= max (would catch typos / argument order issues)
- status is not None and is one of expected values (if not already enforced by type check)

Reply:
Correct. Will add.